Source/Kesmai.Server/Game/Entities/Mobiles/Creatures/Phantasm/SummonedDjinn.cs

```diff
@@ -53,9 +53,9 @@ public SummonedDjinn(ICreatureSpell spell)
 		
 
         var health = (level + (int)magicSkill)* 11 * focusLevel;
-        var defense = (level + 3) * focusLevel;
-		var attack = (level - 3) * focusLevel;
-		var magicResist = (level.Clamp(0,40) * focusLevel);
+        var defense = (level + 3)+(focusLevel * .1);
+		var attack = (level - 3);
```

Comment:
So a level 30 thaum will have 27 attack on a crit. That will not be able to hit much in AG or other higher BG crits. Did you mean to leave off focus level?

Reply:
Until we have a way to share XP from a summoned efreet kill it's kind of a moot point.

Reply:
Chilly approve me!